[BOP-1341] Update authentication (#103)
Mirantis/mke-docs: content/docs/operations/authentication/OIDC.md

are defined in the following table, to configure your chosen OIDC provider.

{{< callout type="info" >}}
For information on how to obtain the field values, refer to the [OIDC provider example](../../../tutorials/authentication-provider-setup/oidc-provider-setup).
{{< /callout >}}

| Field          | Description                                                          |
| -------------- | -------------------------------------------------------------------- |
| `issuer`       | OIDC provider root URL.                                              |
| `clientID`     | ID from the IdP application configuration.                           |
| `clientSecret` | Secret from the IdP application configuration.                       |
| `redirectURI`  | URI to which the provider will return successful authentications to. |

For more information, refer to the official DEX documentation
[OIDC configuration](https://dexidp.io/docs/connectors/oidc/#configuration).

## Test authentication flow

1. Navigate to the MKE dashboard `https://{MKE hostname}`
2. Click **Login** to display the login page.

> 3. Select **Log in with OIDC**. This will redirect you to the OIDC provider's
>    login page.

Author: `Redirect` or `direct` should be used here over `call`

```suggestion
3. Select **Log in with OIDC** to redirect to the login page for your OIDC service provider.
```
